packages/ui/src/composables/useSelectable.ts

```diff
@@ -27,7 +27,7 @@ export const useSelectableProps = {
   ...useStatefulProps,
   ...useLoadingProps,
   ...useValidationProps,
-  arrayValue: { type: null as any as PropType<unknown>, default: null },
+  arrayValue: { type: [String, Boolean, Object, Number] as PropType<any>, default: null },
```

Comment:
```suggestion
  arrayValue: { type: [String, Boolean, Object, Number], default: undefined },
```

Vue can extract this as `PropType<string, boolean, Record<string, any>, number>` automatically.